help/en/html/web/JsonServlet.shtml

```diff
@@ -258,16 +258,10 @@ code {
 					<td>request current state of sensor IS2</td>
 				</tr>
 				<tr>
-					<td><code>{"type":"sensor","data":{"name":"IS2","state":4}}</code></td>
+					<td><code>{"type":"sensor","method":"post","data":{"name":"IS2","state":4}}</code></td>
 					<td><code>{"type":"sensor","data":{"name":"IS2","state":4}}</code></td>
 					<td>set sensor IS2 to InActive (4)</td>
 				</tr>
-				<tr>
-					<td><code>{"type":"turnout","data":{"name":"IT99","state":0}}</code></td>
-					<td><code>{"type":"turnout","data":{"name":"IT99","state":4}}</code></td>
-					<td>sending state=0 is also treated as request for current
-						state</td>
-				</tr>
```

Comment:
Why is this section removed?

Reply:
It's not strictly correct. In a "get", any incoming data values are ignored (except "name"), and the server always returns the current object.
The proper syntax to request the current value is already shown in the first example.

Reply:
For us who are experienced developers, it's easy to take the example for sensors and apply to turnouts. But for newbies, it's much harder. So I think it would be better to keep the turnout example but fix it so it uses "post".

Sensors and turnouts are the most important things that people want to control from JMRI so I think it would be good to have both documented here.

Reply:
OK, I've added a turnout example. I've also updated the responses with the current complete responses.

Reply:
Thank you!